src/styles/searchResults.css

```diff
@@ -0,0 +1,42 @@
+.search-results {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
+
+.no-results {
+    text-align: center;
+    color: white;
+}
+
+
+.card-image {
+    display: flex;
+    flex-wrap: wrap;
```

Comment:
You dont need to use flex here, as theres nothing internal to this component to align. Flex wrap is used on parent components to define how its child components are arranged so its not being used here, this is a [pretty good overview of flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) if youre curious